.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature Request
+about: Use this template for raising a feature request
+labels: 'enhancement'
+title: '[Feature Request]'
+
+---
+
+## Feature request
+
+### Overview
+
+<!-- Provide a high-level description of the feature request. -->
+
+### Specific requirements of feature
+
+<!-- What kind of features are required by the users? -->
+
+### Details
+
+<!--
+Use this section to include any additional information about the feature. If you have a proposal for how to implement this feature, please include it here. For implementation guidelines, please read our contributor guidelines: https://github.com/delta-io/delta/blob/master/CONTRIBUTING.md
+-->
+
+### Motivation
+
+<!-- How will this feature be used? Why is it important? Which users will benefit from it? -->
```

Comment:
Do you think we could move this section to right under "Overview"? Thanks :)

Reply:
Ok.